libraries/AP_EFI/AP_EFI_Serial_Hirth.cpp

```diff
@@ -205,6 +205,7 @@ void AP_EFI_Serial_Hirth::send_request()
 bool AP_EFI_Serial_Hirth::send_target_values(uint16_t thr)
 {
     uint8_t computed_checksum = 0;
+    throttle = 0;
```

Comment:
This is a little too generic a name for a member variable IMO.

Perhaps
```suggestion
    throttle_to_hirth = 0;
```